<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GR Demo (Qt)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="InteractiveGRWidget" name="_gr" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="_btnReset">
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btnPick">
        <property name="text">
         <string>pick</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_chkLogX">
        <property name="text">
         <string>log x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_chkLogY">
        <property name="text">
         <string>log y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_chkGrid">
        <property name="text">
         <string>grid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="_shell"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="_menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="_actionSave"/>
    <addaction name="separator"/>
    <addaction name="_actionQuit"/>
   </widget>
   <addaction name="_menuFile"/>
  </widget>
  <widget class="QStatusBar" name="_statusbar"/>
  <action name="_actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="_actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveGRWidget</class>
   <extends>QWidget</extends>
   <header>qtgr</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
